scripts/7b.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=/opt/nccl/build/lib:/opt/amazon/efa/lib:/opt/amazon/openmpi/lib:/opt/aws-ofi-nccl/lib:/usr/local/cuda/lib:/usr/local/cuda/lib64:/usr/local/cuda:/usr/local/cuda/targets/x86_64-linux/lib/:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/lib:$LD_LIBRARY_PATH
+
+export FI_EFA_SET_CUDA_SYNC_MEMOPS=0
+
+MODEL_ARGS="\
+--model_variant=7b
```

Comment:
@nairbv two reasons for this:
1. beyond model_variant, there are many other differences in them like FSDP vs HSDP, ac vs no-ac, bs, etc. we just put all the $var in a script so we can bette link to our table here https://github.com/foundation-model-stack/fms-fsdp/blob/ceee6f0ed6fdd82b11c293e25205612a18e9af7d/README.md 
2. easier for ppl using other clusters to copy paste.  many/most initial users would be on OCP and they can copy paste the exact same thing inside their yaml file.

Reply:
i think all those other options can be derived from a single model_variant parameter though right?

Reply:
we might have ppl using different hardwares or so, so these are guidance on "what we did so you can use as a reference and optionally adjust correspondingly based on your hardware".

e.g. somehow OCP's A100 will oom with our 7b setup while we were running fine for a few months despite we were also pushing memory to limit (78G/80G).  also, we might be switching to H100 soon (good reminder for me to add a note on that table).   so I would expect users to adjust more values than the variant itself.

Reply:
I mean, if we want maximum customizability, then all of the options should be arguments. if we want standardized ways of doing things, then we should infer from the variant name. We could also infer from the variant except when the other parameters are explicitly overridden.
Having a large number of shell scripts, each of which might need to be customized by the user, seems like it will be harder to maintain. What would we do with these when we get H100s? if we modify these scripts in place we break the scripts for existing users. if we add more scripts like `7b_h100.sh`... it gets messy (and 7b_h100 also could have been an extra option on an existing script).

Reply:
> if we want maximum customizability, then all of the options should be arguments

we are still doing this, we only have one single entrance point which is our [train.sh and train.slurm](https://github.com/foundation-model-stack/fms-fsdp/tree/d385bf25d035bee749a15af46a9da3a0f5b1763a/scripts), and user can either pass arg inside the sh or pass as true command line arg.

All these extra files were merely a companion to the [table](https://github.com/foundation-model-stack/fms-fsdp/blob/ceee6f0ed6fdd82b11c293e25205612a18e9af7d/README.md).  these serves as a purpose of "examples" on "hey, this is what we leveraged train.sh and custmoized to this model variant for our hardware". 

But I get your point, do you think it makes sense if I move all these into an "example" folder rather than the script folder?  the script folder should only have the main entry point.

Reply:
so we have `scripts/` as entry point and `examples/` for a some examples.

I feel we can accept "mess" in the `examples/` folder and let it grow in the future (`7b_h100.sh`) which will serve as a "script zoo". 

`scripts/` should be kept clean and true "customize-able" scripts.

Reply:
I'm still not convinced having a "script zoo" makes sense. they aren't really unique scripts, just configurations of a single script

Reply:
Ok, then I think table itself might be enough then.  One can definitely use the table and "customize" the script to generate all these examples.